src/core/MdComponent.js

```diff
@@ -1,4 +1,4 @@
-import MdTheme from 'core/MdTheme'
+import MdTheme from './MdTheme'
```

Comment:
That's alias
https://github.com/vuematerial/vue-material/blob/44b0147c5c9d4c541046c0b06f99d055ad13a5a5/build/config.js#L36

Reply:
Yes but when one wants to use the component on it's own (importing it when creating new components outside) it is throwing error.